test/FunctionalTests/Taupo/Source/Taupo.Astoria/Client/ClientResponseQueryValueBuilder.cs

```diff
@@ -402,6 +402,21 @@ private QueryStructuralValue BuildFromEntityInstance(EntityInstance instance, Qu
             return entity;
         }
 
+        /// <summary>
+        /// Builds an instance of QueryPrimitiveValue from an primitiveValue instance.

```

Comment:
>QueryPrimitiveValue [](start = 34, length = 19)

QueryScalarValue

Reply:
fixed

---
In reply to: [289951744](https://github.com/OData/odata.net/pull/1474#discussion_r289951744) [](ancestors = 289951744)